### DISCUSSION TITLE
How to get data from command app_status or information staged: 6774/6774 MB 100%, downloaded: 3104/3104 MB 100% - 0 KB/s

### DISCUSSION THREAD

**Discussion by L4legenda** [ValvePython/steam#416] [Q&A] [answered]
command in steam console 'app_status <id>' output line 
update started: Sat Nov 19 21:03:22 2022, staged: 6774/6774 MB 100%, downloaded: 3104/3104 MB 100% - 0 KB/s

How to get data ?

**Comment by rossengeorgiev** [comment, marked as answer]
This package doesn't interact with the steam desktop client, so you won't be able to get that information.
